snap/epoch_test.go

```diff
@@ -131,6 +131,67 @@ func (s epochSuite) TestGoodEpochs(c *check.C) {
 	}
 }
 
+func (s epochSuite) TestGoodEpochsInSnapYAML(c *check.C) {
+	defer snap.MockSanitizePlugsSlots(func(snapInfo *snap.Info) {})()
+
+	type Tt struct {
+		s string
+		e snap.Epoch
+	}
+
+	tests := []Tt{
+		{s: `epoch: 0`, e: snap.Epoch{Read: []uint32{0}, Write: []uint32{0}}},
```

Comment:
Since you're checking this for JSON, why not add it for YAML too:
```
{s: `epoch: null`, e: snap.Epoch{Read: []uint32{0}, Write: []uint32{0}}},
```